meinberlin/apps/projects/signals.py

```diff
@@ -1,49 +1,35 @@
-from django.core.cache import cache
 from django.db.models.signals import post_delete
 from django.db.models.signals import post_save
 from django.dispatch import receiver
 
 from adhocracy4.dashboard import signals as a4dashboard_signals
+from adhocracy4.phases.models import Phase
 from adhocracy4.projects.models import Project
-from meinberlin.apps.projects.tasks import get_next_projects_end
-from meinberlin.apps.projects.tasks import get_next_projects_start
+from meinberlin.apps.projects.tasks import set_cache_for_projects
 
 
 @receiver(a4dashboard_signals.project_created)
 @receiver(a4dashboard_signals.project_published)
 @receiver(a4dashboard_signals.project_unpublished)
 def post_dashboard_signal_delete(sender, project, user, **kwargs):
-    cache.delete_many(
-        [
-            "projects_activeParticipation",
-            "projects_futureParticipation",
-            "projects_pastParticipation",
-            "private_projects",
-            "extprojects",
-        ]
-    )
+    """Refresh project, plan and extproject cache on dashboard signal"""
+    set_cache_for_projects.delay_on_commit(get_next_projects=True)
+
+
+@receiver(post_save, sender=Phase)
+@receiver(post_delete, sender=Phase)
+def post_phase_save_delete(sender, instance, **kwargs):
```

Comment:
added a signal for phases (e.g. when a future project gets a new phase which turns it into an active project)